policyengine_us/parameters/gov/states/la/tax/income/deductions/itemized/medical_expenses/exceedance/limit_to_medical_expense_deduction.yaml

```diff
@@ -0,0 +1,16 @@
+description: Louisiana subtracts from adjusted gross income the excess of the federal medical expense deduction over the standard deduction if this is true.
```

Comment:
@MaxGhenis  is there a better yaml file name?

Reply:
This is putting the cart before the horse. It's not clear what the policy does, from either the yaml or Python files. Also as I wrote in a previous review, `exceedance` is not a word - use `excess`.